src/components/guideLineLayer.ts

```diff
@@ -0,0 +1,180 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Components {
+  enum PropertyMode { value, pixel };
```

Comment:
To my understanding, enum's are not variable, and thus so to not confuse them with other variables, they should not be undercased